DMRPP test files in no longer available?

I tried running tests locally and got the following error:

```python
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=4, connect=0, read=None, redirect=None, status=2)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x15110ef30>: Failed to establish a new connection: [Errno 61] Connection refused')': /opendap/dmrpp_test_files/ATL03_20181228015957_13810110_003_01.2var.h5.dmrpp.dmr
================================================== short test summary info ===================================================
FAILED src/pydap/tests/test_open_dap4_url.py::test_groups - requests.exceptions.HTTPError: HTTP Error occurred 404 Client Error: 404 for url: http://test.opendap.org/opendap/dmrpp_t...
```


I also tried nagivating to the test folder on my browser and got a 403. 

### solution
It is possible this will no longer be available, which is fine. I can try a different approach.

For now I will skip this test

--- a/src/pydap/tests/test_open_dap4_url.py
+++ b/src/pydap/tests/test_open_dap4_url.py
@@ -20,7 +20,7 @@ def test_coads():
     pydap_ds["COADSX"][10:12:1]
 
 
-@pytest.mark.skip(reason="path temporarily unavailable")
+@pytest.mark.skip(reason="path temporarily unavailable. See OPENDAP/bes#1332")
 def test_groups():
     url = base_url + "/opendap/dmrpp_test_files/"
     pydap_ds = open_url(url + "ATL03_20181228015957_13810110_003_01.2var.h5.dmrpp")

--- a/src/pydap/tests/test_open_dap4_url.py
+++ b/src/pydap/tests/test_open_dap4_url.py
@@ -20,6 +20,7 @@ def test_coads():
     pydap_ds["COADSX"][10:12:1]
 
 
+@pytest.mark.skip(reason="path temporarily unavailable")
 def test_groups():
     url = base_url + "/opendap/dmrpp_test_files/"
     pydap_ds = open_url(url + "ATL03_20181228015957_13810110_003_01.2var.h5.dmrpp")

